debug-cli/scripts/overwrite-key/README.md

```diff
@@ -0,0 +1,345 @@
+# Camunda Key Recovery Procedure
+
+Recovery procedure for fixing corrupted key values in Camunda partitions.
+
+## Prerequisites
+
+- Kubectl access to the cluster
+- `jq` installed for JSON parsing
+- **Cold backup of all broker data** (PVC snapshots or volume backups)
+- Cluster must be shut down during recovery
+
+## Required Information
+
+Before starting, you need:
+- **NAMESPACE**: Kubernetes namespace where Camunda is deployed
+- **PARTITION_ID**: The partition ID to recover
+- **NEW_KEY**: New key value to set
+- **NEW_MAX_KEY**: New maximum key value (optional, but recommended)
+- **LAST_LEADER_BROKER**: The broker ID that was leader for this partition
+- **PARTITION_BROKER_IDS**: Space-separated list of broker IDs hosting this partition (e.g., "0 1 2")
+- **CONTAINER_IMAGE**: The Camunda container image (e.g., "camunda/camunda:8.6.1")
+- **SNAPSHOT_ID**: Specific snapshot to use (optional - required only if multiple snapshots exist)
+
+**Tip:** Use `../detect-config.sh <namespace>` to auto-detect most of these values (namespace, broker count, pod prefix, PVC prefix, container image, etc.)
```

Comment:
❓ Why is this not built in? Can't we integrate this in the script?

Reply:
I felt like this is a more general script and I'd like the script to be completely "fixable" by a human if for some reason it does not work. 
As a result, you can actually just use the `generate-job` script exporting the right ENV var.